Projects/UOContent/Multis/Houses/HouseFoundation.cs

```diff
@@ -1156,9 +1156,7 @@ public void EndConfirmCommit(Mobile from)
                 if (from.AccessLevel >= AccessLevel.GameMaster && cost != 0)
                 {
                     from.SendMessage(
-                        "{0} gold would have been {1} your bank if you were not a GM.",
-                        cost.ToString(),
-                        cost > 0 ? "withdrawn from" : "deposited into"
+                        $"{cost.ToString()} gold would have been {(cost > 0 ? "withdrawn from" : "deposited into")} your bank if you were not a GM."
```

Comment:
Let's remove the `.ToString()` from inside the interpolations. Not only is it syntactically redundant but .net core will use TryFormat instead if the value is a primitive or ISpanFormattable which is much more optimized.

Reply:
Done.
Additional thanks for the inside information